doc/tntcxx_getting_started.rst

```diff
@@ -0,0 +1,524 @@
+
+Connecting to Tarantool from C++
+=================================
+
+To simplify the start of your working with the Tarantool C++ connector, we will
+use the `example application <https://github.com/tarantool/tntcxx/blob/master/examples/Simple.cpp>`_
+from the connector repository. We will go step by step through the application
+code and explain what each part does.
+
+The following main topics are discussed in this manual:
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _gs_cxx_prereq:
+
+Pre-requisites
+----------------
+
+To go through this Getting Started exercise, we need the following
+pre-requisites to be done:
+
+* :ref:`installing and building the Tarantool C++ connector <gs_cxx_prereq_install>`
+* :ref:`starting Tarantool and creating a database <gs_cxx_prereq_tnt_run>`
+* :ref:`seting up access rights <gs_cxx_prereq_access>`.
+
+.. _gs_cxx_prereq_install:
+
+Installation
+~~~~~~~~~~~~~
+
+Currently supported OS for the Tarantool C++ connector usage is Linux.
```

Comment:
1. There's no packages to build at all
2. MacOS and Windows are definitely not supported yet. As for other unix-like distributives - I can't say for sure since haven't tested it
3. Of course

Reply:
"Supported" means that we can install, build and use the connector on Linux only. It doesn't work on other OS. We don't build packages to download&install if you mean that; currently the connector can only be build from source. Didn't fully get the point of fixing bugs on Linux. Can you please elaborate?